src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -98,7 +98,7 @@ def parse_function_role_prompt(function_str):
 
 def parse_chat(chat_str):
     # openai chat api only supports below roles.
-    separator = r"(?i)\n+\s*(system|user|assistant|function)\s*:\s*\n"
+    separator = r"(?i)\n+\s*#?\s*(system|user|assistant|function)\s*:\s*\n"
```

Comment:
add some comments here to explain two things:
- `#` usage
- backward compatibility.